charts/posit-chronicle/templates/stateful-set.yaml

```
@@ -1,47 +1,65 @@
 ---
+{{- $root := . }}
 apiVersion: apps/v1
 kind: StatefulSet
 metadata:
-  name: chronicle-server
-  namespace: {{ $.Release.Namespace }}
+  name: {{ include "posit-chronicle.fullname" . }}
+  namespace: {{ include "posit-chronicle.namespace" . }}
+  labels:
+    {{- include "posit-chronicle.labels" . | nindent 4 }}
+  annotations:
+    {{- with .Values.commonAnnotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
 spec:
-  serviceName: chronicle-server
   replicas: {{ .Values.replicas }}
+  serviceName: {{ include "posit-chronicle.fullname" . }}
   selector:
     matchLabels:
-      {{- include "posit-chronicle.pod.selectorLabels" . | trim | nindent 6 }}
+      {{- include "posit-chronicle.selectorLabels" . | trim | nindent 6 }}
   template:
     metadata:
       labels:
-        {{- include "posit-chronicle.pod.labels" . | trim |  nindent 8 }}
-        {{- include "posit-chronicle.pod.selectorLabels" . | trim |  nindent 8 }}
+        {{- include "posit-chronicle.labels" . | trim |  nindent 8 }}
+        {{- with .Values.pod.labels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
       annotations:
-        {{- include "posit-chronicle.pod.annotations" . | trim |  nindent 8 }}
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+        {{- include "posit-chronicle.pod.annotations" . | trim | nindent 8) }}
```

```suggestion
        {{- include "posit-chronicle.pod.annotations" . | trim | nindent 8 }}
```

I was trying to fiddle around with the latest locally and came across this